porcupine/plugins/run/terminal.py

```diff
@@ -37,8 +41,13 @@ def _run_in_windows_cmd(command: str, cwd: Path, env: dict[str, str]) -> None:
     subprocess.Popen(real_command, env=env)
 
 
-def _run_in_macos_terminal_app(command: str, cwd: Path, env: dict[str, str]) -> None:
+def _run_in_macos_terminal_app(command: str | None, cwd: Path, env: dict[str, str]) -> None:
     log.debug("using MacOS terminal.app")
+
+    if not command:
+        subprocess.Popen(["open", "-a", "Terminal", str(cwd)], env=env)
```

Comment:
This is how you launch a script: `open -a Terminal <scriptfile>`. Does the same command work for directories as well? I don't have a mac so I can't test this myself, and I also failed googling this.